### PR TITLE
199 migrate to solana sdk 1.6.9

### PR DESCRIPTION
**Pull request by otselnik**

### CODE REVIEW

File: evm_loader/docker-compose-test.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: "2.1"
 services:
   solana:
     container_name: solana
-    image: cybercoredev/solana:v1.6.7-resources
+    image: cybercoredev/solana:9e68ded325b8ceb25d346cd6e2656f790ba89033
```
Comment (s-medvedev) [inline evm_loader/docker-compose-test.yml]: Use tagged image here (not commit hash).

File: Dockerfile [hunk 2]
```diff
@@ -41,7 +41,7 @@ RUN solc --output-dir . --bin *.sol && \
         ls -l
 
 # Define solana-image that contains utility
-FROM cybercoredev/solana:v1.6.7-resources AS solana
+FROM cybercoredev/solana:9e68ded325b8ceb25d346cd6e2656f790ba89033 AS solana
```
Comment (s-medvedev) [inline Dockerfile]: Use tagged image here (not commit hash).

File: evm_loader/docker-compose-test.yml (hunk 1 above)
Comment (otselnik) [reply evm_loader/docker-compose-test.yml]: fixed

File: Dockerfile (hunk 2 above)
Comment (otselnik) [reply Dockerfile]: fixed

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**